manifests/init.pp

```diff
@@ -577,7 +577,7 @@
     if $manage_pcsd_auth and $is_auth_node {
       # TODO - verify if this breaks out of the sensitivity
       $hacluster_password = $sensitive_hacluster_password.unwrap
-      $auth_credential_string = "-u hacluster -p ${hacluster_password}"
+      $auth_credential_string = "-u hacluster -p \'${hacluster_password}\'"
```

Comment:
if the string is wrapped in single quotes, it should not be required to escape them?

Reply:
In my testing, I found that the escape character is required.